README.md

```diff
@@ -7,100 +9,92 @@
 [![Dependabot Status](https://api.dependabot.com/badges/status?host=github&repo=peterhuene/azure-functions-rs)](https://dependabot.com)
 [![license](https://img.shields.io/crates/l/azure-functions.svg)](https://github.com/peterhuene/azure-functions-rs/blob/master/LICENSE)
 
-This is an early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
+An early-stage framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
```

Comment:
I think it's time to get rid of `early-stage`.  While we're not yet a 1.0 stable release, we're far from early :smile:

Reply:
```suggestion
A framework for implementing [Azure Functions](https://azure.microsoft.com/en-us/services/functions/)
```